Allow check for pluralRules if count is zero

groomain opened the pull request:
It allow to use the plural rule _zero

Fix https://github.com/vinissimus/next-translate/issues/796
And https://github.com/vinissimus/next-translate/issues/771

aralroca reviewed (changes requested):
Great!  Would you add a test with _zero? so we make sure it doesn't break in the future. Thanks!

groomain commented:
I've added tests and changed some.

In fr => count 0 should be _one
In ar => count 0 should be _zero
In en => count 0 should be _other

https://english.stackexchange.com/questions/13073/correct-plural-form-of-a-noun-preceded-by-zero#:~:text=In%20English%2C%20every%20number%20that,password%20expires%20in%200%20days.

aralroca commented:
@all-contributors please add @groomain for code
